### PR TITLE
Added back in the functions to create rss torrent providers from it's 'csv' data string

### LINKED ISSUE #3130: Custom torrent providers information lost after restart
After restart **RSS URL** and **Cookies (optional)** are missing from the custom provider. Latest develop.

### PR DESCRIPTION
**Pull request by p0psicles**
..This is needed for a proper migration.

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

Should fix #3130

### CODE REVIEW

File: medusa/config.py [hunk 1]
```diff
@@ -1104,6 +1104,68 @@ def make_id(name):
...
+                elif len(values) == 10:
+                    name, url, cookies, title_tag, enabled, search_mode, search_fallback, enable_daily, enable_backlog, enable_manualsearch = values
+                elif len(values) == 8:
+                    name, url, cookies, enabled, search_mode, search_fallback, enable_daily, enable_backlog = values
+                else:
+                    enabled = values[4]
+                    name = values[0]
+                    url = values[1]
+            except ValueError:
+                log.error('Skipping RSS Torrent provider string: {0}, incorrect format', config)
```
Comment (p0psicles) [inline medusa/config.py]: Missed that. I'll migrate the entire module to new style logger in this pr.

**Comment by p0psicles** [comment]
@medariox can you test this branch?

**Comment by medariox** [comment]
@p0psicles 
Will switch branch tomorrow and let you know.

**Comment by medariox** [comment]
This fixes the migration issue, but it's not exactly what I meant in #3130 (I will add some more info there). Nonetheless an important fix, thank you!

**Pull request merged**